docs/common-coreboot-docs/dasharo_tools_suite.md

```diff
@@ -110,6 +110,28 @@ To access Dasharo Tools Suite:
 * in the Network Boot menu, select the `Dasharo Tools Suite` option,
 * the DTS menu will now appear.
 
+## Common requirements - disabling Secure Boot
+
+Any procedure which affects the firmware should be preceded by controlling
+the Secure Boot status and if it is turned on, turning it off. The enabled
+Secure boot will not only prevent you f rom operating on the firmware, but
+you will also not be able to launch DTS.
+
+To check the Secure Boot state:
+
+1. Turn off the station, on which you want to test the Dasharo firmware.
+1. Turn the station on and go to the next step immediately.
+1. Hold the `BIOS SETUP KEY` to enter the `BIOS MENU`.
+1. Enter the `Administer Secure Boot` menu using the arrow keys and Enter.
```

Comment:
@sulewskiprzemyslaw it is called this way on NV41xx? What if we use other stock fw on different machine? Maybe lets say here that we need to localize option for Secure Boot.

Reply:
@TomaszAIR improved. I changed this point to: `Localise and enter the Secure Boot menu using the arrow keys and Enter`